# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# T2 SDE: package/.../alternatives/Makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- alternatives-0.2.0rc3/Makefile	2004-10-08 16:47:45.000000000 +0200
+++ alternatives-patched/Makefile	2005-04-07 11:47:19.000000000 +0200
@@ -30,6 +30,7 @@
 DATADIR=/usr/share/alternatives
 MANDIR = /usr/share/man
 MAN1DIR = $(MANDIR)/man1
+ALTDIR = /etc/alternatives
 
 HELP2MAN1 = help2man -N -s1
 INSTALL = install
@@ -55,12 +56,12 @@
 	$(HELP2MAN1) -i doc/$@.inc ./$< >$@
 
 
-install: all install-bin install-compat install-lib install-man
+install: all install-bin install-compat install-lib install-man install-cfg
 
 install-bin:
 	$(MKDIR_P) -m755 $(DESTDIR)$(BINDIR)
-	install -pm 755 $(PROGRAMS:%=alternatives-%) $(DESTDIR)$(BINDIR)
-	install -pm 755 alternatives-helper $(DESTDIR)/$(BINDIR)
+	$(INSTALL) -p -m755 $(PROGRAMS:%=alternatives-%) $(DESTDIR)$(BINDIR)
+	$(INSTALL) -p -m755 alternatives-helper $(DESTDIR)/$(BINDIR)
 
 install-compat:
 	$(MKDIR_P) -m755 $(DESTDIR)$(BINDIR) $(DESTDIR)$(SBINDIR)
@@ -69,13 +70,16 @@
 
 install-lib:
 	$(MKDIR_P) -m755 $(DESTDIR)$(DATADIR)
-	install -pm755 $(LIBRARY) $(DESTDIR)$(DATADIR)
+	$(INSTALL) -p -m755 $(LIBRARY) $(DESTDIR)$(DATADIR)
 	
 
 install-man:
 	$(MKDIR_P) -m755 $(DESTDIR)$(MAN1DIR)
 	$(INSTALL) -p -m644 $(MAN1PAGES) $(DESTDIR)$(MAN1DIR)/
 
+install-cfg:
+	$(INSTALL) -d $(ALTDIR)/{auto,packages.d,links}
+
 clean:
 	$(RM) $(TARGETS) *~
 
